2020/2020-04-14.committee.md

```diff
@@ -0,0 +1,210 @@
+# UWA PCS Committee Meeting Agenda 2020-04-14
```

Comment:
Consider adding "Opening" section for opening time and attendance for good document structure?

Reply:
Good idea